compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DataflowAnalyzer.java

```diff
@@ -357,6 +357,14 @@ private void checkForUninitializedGlobalVars(List<BLangVariable> globalVars) {
                 case TUPLE_VARIABLE:
                     checkForUninitializedGlobalVars(((BLangTupleVariable) globalVar).memberVariables);
                     break;
+                case RECORD_VARIABLE:
```

Comment:
Don't we have to handle the ERROR_VARIABLE scenario here?

Reply:
Since we have to handle this from the parser and remove these things anyway I didn't implement it. Will start working on it right after this is merged

Reply:
Ack.